decidim-core/app/assets/stylesheets/decidim/modules/_cards.scss

```diff
@@ -730,9 +731,39 @@ $datetime-bg: $primary;
   font-size: $small-font-size;
   font-weight: bold;
   position: absolute;
-  background-color: $card-bg;
-  margin-top: -$global-margin * .75;
+  background-color: transparent;
+  text-shadow:
+    -1px -1px 0 $card-bg,
+    1px -1px 0 $card-bg,
+    -1px 1px 0 $card-bg,
+    1px 1px 0 $card-bg;
+  margin-top: ($global-margin * .6) * -1;
   margin-left: $card-padding-small;
+  z-index: 0;
+
+  &::before{
+    content: " ";
+    height: $border-width;
+    display: inline-block;
+    background: $card-bg;
+    position: absolute;
+    left: 0;
+    right: 0;
+    z-index: -1;
+    top: $global-margin * .5;
+  }
+
+  .success &,
```

Comment:
This snippet compiles to:
```css
.success .card__label, .muted .card__label, .warning .card__label, .alert .card__label {
  margin-top: -0.95rem;
}
.success .card__label::before, .muted .card__label::before, .warning .card__label::before, .alert .card__label::before {
  height: 8px;
  top: 0.45rem;
}
```
That means it loses the card scope. It would so odd, but possible, that any success/alert/whatever status applies these styles to a `.card__label` class.
So, to solve this out, prepend the `.card` class. In that way:
```css
.card.success &,
.card.muted &,
.card.warning &,
.card.alert & {
   // properties
}
```
Now we're guaranteeing that the card status, and only that status, is which applies those styles.

Reply:
Yes, good point, I'll add the `.card` scope